mdformat_myst/plugin.py

```diff
@@ -40,11 +44,6 @@ def update_mdit(mdit: MarkdownIt) -> None:
     # Enable dollarmath markdown-it extension
     mdit.use(dollarmath_plugin)
 
-    # Enable footnote markdown-it extension
-    mdit.use(footnote_plugin)
-    # MyST has inline footnotes disabled
-    mdit.disable("footnote_inline")
```

Comment:
Out of interest, does mdformat-footnote have `footnote_inline` enabled?

Reply:
Nope, it doesn't: https://github.com/executablebooks/mdformat-footnote/blob/80852fc20cfba7fd0330b9ac7a1a4df983542942/mdformat_footnote/plugin.py#L17